src/base/views/viewer.tsx

```diff
@@ -211,9 +218,9 @@ export class HiddenModelViewer implements IViewer {
     @inject(TYPES.ILogger) protected logger: ILogger;
 
     constructor(@inject(TYPES.ModelRendererFactory) modelRendererFactory: ModelRendererFactory,
-                @inject(TYPES.PatcherProvider) patcherProvider: PatcherProvider,
-                @multiInject(TYPES.HiddenVNodePostprocessor) @optional() hiddenPostprocessors: IVNodePostprocessor[]) {
-        this.hiddenRenderer = modelRendererFactory('hidden', hiddenPostprocessors);
+        @inject(TYPES.PatcherProvider) patcherProvider: PatcherProvider,
+        @multiInject(TYPES.HiddenVNodePostprocessor) @optional() hiddenPostprocessors: IVNodePostprocessor[]) {
+        this.hiddenRenderer = modelRendererFactory('hidden', hiddenPostprocessors, { root: true, hidden: true });
```

Comment:
Do you use the `root` and `hidden` properties anywhere?
`hidden` seems redundant to `targetKind`.

Reply:
No, I don't use them. I just thought they may be good to have, but this was rather speculative and you are right, that this is actually redundant to `targetKind`.

Reply:
Isn't `args.root === true` equivalent to `args. parentArgs === undefined`? In that case we could remove `root` as well.

Reply:
Right, I just put it in there to avoid having an empty root args object. But it doesn't add real value per se. So you're right, we can remove it.